_includes/markdown/PHP.md

```diff
@@ -616,6 +616,51 @@ if ( ! empty( $_POST['_wpnonce'] ) && wp_verify_nonce( $_POST['_wpnonce'], 'my_a
 ?>
 ```
 
+#### Internationalization
+
+All text strings in a project should be internationalized using core localization functions. Even if the project does not currently dictate a need for translatable strings, this practice ensures translation-readiness should a future need arise.
+
+WordPress provides a myriad of [localization functionality](https://codex.wordpress.org/I18n_for_WordPress_Developers). Engineers should familiarize themselves with features such as [pluralization](https://codex.wordpress.org/I18n_for_WordPress_Developers#Plurals) and [disambiguation](https://codex.wordpress.org/I18n_for_WordPress_Developers#Disambiguation_by_context) so translations are flexible and translators have the information they need to work accurately.
+
+Samuel Wood (Otto) put together a guide to WordPress internationalization best practices, and engineers should take time to familiarize themselves with its guidance: [Internationalization: You’re probably doing it wrong](http://ottopress.com/2012/internationalization-youre-probably-doing-it-wrong/)
+
+Localizing a project differs from the core approach in two distinct ways:
+* A unique text domain should be used with all localization functions
+* Internationalized output should always be escaped
+##### Text Domains
+
+Each project should leverage a unique text domain for its strings. Text domains should be lowercase, alphanumeric, and use hyphens to separate multiple words: `tenup-project-name`.
+
+Text domains should never be stored in a variable or named constant when used with core localization functions, as this practice can often produce unexpected results. Most tools used to create translation rely on [GNU gettext](https://www.gnu.org/software/gettext/) scanning source code for translation functions. PHP code won't be interpreted, only scanned like it was a block of plain text, and translation tools won't be able to assign the text domain correctly if it's not there in plain text.
+
+```php
+// Always this
+$string = __('Hello World', 'plugin-domain');
+// Never this
+$string = __('Hello World', $plugin_domain);
```

Comment:
I think both. Not having the string literal on either side can break POEdit parsing (it can't find the string to translate or it can't find the textdomain to use in translation).